l3kernel/l3color.dtx

```diff
@@ -52,6 +49,32 @@
 %
 % \begin{documentation}
 %
+% \section{Color in boxes}
+%
+% Controlling the color of text in boxes requires a small number of control
+% functions, so that the boxed material uses the color at the point where
+% it is set, rather than where it is used.
+%
+% \begin{function}[added = 2011-09-03]{\color_group_begin:, \color_group_end:}
+%   \begin{syntax}
+%     \cs{color_group_begin:}
+%       \ldots
+%     \cs{color_group_end:}
```

Comment:
You mean because one could use either `\color@begingroup` or `\color_group_begin:`? The definitions themselves are very light, so it's just a matter of semantics - in an `expl3` context I'd expect the definition here, in classical code the one from the `color` package. Or were you thinking something else?

Reply:
I mean that with https://github.com/latex3/latex2e/issues/488 we will end up with 
```
\let\color@begingroup\begingroup
\def\color@endgroup{\endgraf\endgroup}
```
and should perhaps make this identical on both sides (also needs thinking how to deal with the \endgraf).

Reply:
Ah, right: the `expl3` interfaces for `vbox`, etc., all have the `\par` in the box itself. I guess we should standardise, yes.

Reply:
I think we should (with the \par inside the color endgroup) and it would need to be  `\par` or `\endgraf` not `\tex_par:D` to allow for  `\par` redefinitions for paragraph tagging.

Reply:
That way it also only applies to vertical-mode boxes. If we want something explicitly about tagging, shouldn't it be named semantically?